frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx

```diff
@@ -69,12 +66,13 @@ export const VMUsersList: React.FC<VMUsersListProps> = ({ vmi, vmStatusBundle, d
   const guestAgentInfo = new GuestAgentInfoWrapper(guestAgentInfoRaw);
   const userList = guestAgentInfo.getUserList();
 
-  if (vmStatusBundle.status !== VMStatus.RUNNING) {
-    return <div className="text-center">{VIRTUAL_MACHINE_IS_NOT_RUNNING}</div>;
-  }
+  const guestAgentFieldNotAvailMsg = getGuestAgentFieldNotAvailMsg(
+    isGuestAgentInstalled(vmi),
+    vmStatusBundle.status,
+  );
 
-  if (!isGuestAgentInstalled(vmi)) {
-    return <StatusItem Icon={BlueInfoCircleIcon} message={NO_GUEST_AGENT_MESSAGE} />;
+  if (!guestAgentInfoRaw && guestAgentFieldNotAvailMsg) {
```

Comment:
`guestAgentFieldNotAvailMsg` is redundant - it will always have a value no ?

Reply:
Right :exclamation: fixing

Reply:
Thanks :+1: fixed